src/ems_client.js

```diff
@@ -104,11 +106,16 @@ export class EMSClient {
     proxyPath,
   }) {
 
+    // Remove kbnVersion in 8.0
+    if (kbnVersion) {
+      console.warn('The "kbnVersion" parameter for ems-client is deprecated. Please use "appVersion" instead.');
```

Comment:
thx. I guess we'll need corresponding Kibana update then as well?

Reply:
> thx. I guess we'll need corresponding Kibana update then as well?

Yup. This will be part of the changes we need when we bump the ems-client dependency in Kibana and ems-landing-page for 7.7.